sailor/assetcentral/equipment.py

```diff
@@ -180,7 +180,7 @@ def find_workorders(self, *, extended_filters=(), **kwargs) -> WorkorderSet:
 
     def get_indicator_data(self, start: Union[str, pd.Timestamp, datetime.timestamp, datetime.date],
                            end: Union[str, pd.Timestamp, datetime.timestamp, datetime.date],
-                           indicator_set: IndicatorSet = None) -> TimeseriesDataset:
+                           indicator_set: IndicatorSet = None) -> 'TimeseriesDataset':
```

Comment:
pretty sure there's no need for these quotes, you already have a `from __future__ import annotations` at the top